### PR TITLE
Mute command

### PR DESCRIPTION
**Pull request by Will3126**
• Adds muted role to a user (mention or id) with reason
• Changes channel perms for all channels for muted role
• Removes muted role to a user (mention or id)

### CODE REVIEW

**Comment by Will3126** [comment]
Not yet tested, going to test on Friday.

File: src/bot/commands/mute.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        Discord.Permissions.ADD_REACTIONS,
+        Discord.Permissions.ATTACH_FILES,
+        Discord.Permissions.EMBED_LINKS,
+        Discord.Permissions.CONNECT,
+        Discord.Permissions.SPEAK,
+        Discord.Permissions.STREAM,
+        Discord.Permissions.USE_VAD,
+        Discord.Permissions.MOVE_MEMBERS,
+        Discord.Permissions.MUTE_MEMBERS,
+        Discord.Permissions.DEAFEN_MEMBERS,
```
Comment (MidSpike) [inline src/bot/commands/mute.js:31]: this is a bug.
all permissions should start with `Discord.Permissions.FLAGS.`
most are missing this.

File: src/bot/commands/umute.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        const member_lookup_query = message.mentions.members.first()?.id ?? command_args[0];
+        const member = message.guild.members.resolve(member_lookup_query);
+        const unmute_reason = command_args.slice(1).join(' ').trim();
+
+        /* Remove muted role to member */
+        try {
+            await member.roles.remove(muted_users_role_id, unmute_reason);
+        } catch (error) {
+            console.trace(error);
+            await message.reply('Failed to remove the muted role to the user!').catch(console.warn);
```
Comment (MidSpike) [inline src/bot/commands/umute.js:31]: Check your grammar and choice of words

**Review by MidSpike — CHANGES_REQUESTED** [review]

File: src/bot/commands/umute.js (hunk 2 above)
Comment (Will3126) [reply src/bot/commands/umute.js:31]: Just line 31

**Comment by Will3126** [comment]
changes made testing mostly done waiting on reviews

**Review by MidSpike — APPROVED** [review]
Look's good so far

**Review by rossloika — APPROVED** [review]

**Comment by MidSpike** [comment]
Merging without @itsdrawn 's approval.
He is taking too long.

**Pull request merged**